sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/Schema.java

```diff
@@ -142,82 +142,163 @@ public Builder addByteField(String name) {
       return this;
     }
 
+    public Builder addNullableByteField(String name) {
+      fields.add(Field.nullable(name, FieldType.BYTE));
```

Comment:
Probably we should use the existing "addNullableField" function when implementing these.

Reply:
sure, no problem